ros_bt_py/ros_bt_py/ros_nodes/action.py

```diff
@@ -498,7 +498,10 @@ class Action(Leaf):
     _goal_type: type
     _feedback_type: type
     _result_type: type
-    _ac: Optional[Client] = None
+    _ac: Optional[ActionClient] = None
+    _feedback = None
+    _internal_state = ActionStates.IDLE
+    passthrough: bool = False
```

Comment:
I am not quite sure what this does

Reply:
Not read so I removed it. Everything still works, so it cannot have been that important.

Reply:
Famous last words :D